File: MCP服务器/security/wso2_open-mcp-auth-proxy_081f010_go/internal/logging/logger.go
package logger

import (
	"log"
)

var isDebug = false

// SetDebug enables or disables debug logging
func SetDebug(debug bool) {
	isDebug = debug
}

// Debug logs a debug-level message
func Debug(format string, v ...interface{}) {
	if isDebug {
		log.Printf("DEBUG: "+format, v...)
	}
}

// Info logs an info-level message
func Info(format string, v ...interface{}) {
	log.Printf("INFO: "+format, v...)
}

// Warn logs a warning-level message
func Warn(format string, v ...interface{}) {
	log.Printf("WARN: "+format, v...)
}

// Error logs an error-level message
func Error(format string, v ...interface{}) {
	log.Printf("ERROR: "+format, v...)
}

File: reference-mcp-servers/Communication/gitmotion_ntfy-me-mcp_7bcd80b_typescript/src/utils/logger.ts
export class Logger {
  private static instance: Logger;

  private constructor() {}

  public static getInstance(): Logger {
    if (!Logger.instance) {
      Logger.instance = new Logger();
    }
    return Logger.instance;
  }

  public info(message: string): void {
    console.error(`[NTFYME-LOG-INFO]: ${message}`);
  }

  public warn(message: string): void {
    console.error(`[NTFYME-LOG-WARNING]: ${message}`);
  }

  public error(message: string): void {
    console.error(`[NTFYME-LOG-ERROR]: ${message}`);
  }
}
